src/ResourceManager/Search/Commands.Management.Search/Commands.Management.Search.csproj

```diff
@@ -0,0 +1,106 @@
+    <DelaySign>true</DelaySign>
+    <ErrorReport>prompt</ErrorReport>
+    <WarningLevel>4</WarningLevel>
+  </PropertyGroup>
+  <PropertyGroup>
+    <StartupObject />
+  </PropertyGroup>
+  <ItemGroup>
+    <None Include="AzureRM.Search.psd1">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
+    </None>
+    <None Include="MSSharedLibKey.snk" />
+    <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <Compile Include="Models\PSSearchAdminKey.cs" />
+    <Compile Include="Models\PSSearchQueryKey.cs" />
+    <Compile Include="Models\PSSearchService.cs" />
+    <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Properties\Resources.Designer.cs">
+      <AutoGen>True</AutoGen>
+      <DesignTime>True</DesignTime>
+      <DependentUpon>Resources.resx</DependentUpon>
+    </Compile>
+    <Compile Include="SearchService\GetSearchServiceAdminKeyPairCommand.cs" />
+    <Compile Include="SearchService\GetSearchServiceCommand.cs" />
+    <Compile Include="SearchService\GetSearchServiceQueryKeyCommand.cs" />
+    <Compile Include="SearchService\NewSearchServiceAdminKeyCommand.cs" />
+    <Compile Include="SearchService\NewSearchServiceCommand.cs" />
+    <Compile Include="SearchService\NewSearchServiceQueryKeyCommand.cs" />
+    <Compile Include="SearchService\RemoveSearchServiceCommand.cs" />
+    <Compile Include="SearchService\RemoveSearchServiceQueryKey.cs" />
+    <Compile Include="SearchService\SearchManagementClient.cs" />
+    <Compile Include="SearchService\SearchServiceBaseCmdlet.cs" />
+    <Compile Include="SearchService\SetSearchServiceCommand.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\..\Common\Commands.Common.Authentication.Abstractions\Commands.Common.Authentication.Abstractions.csproj">
+      <Project>{70527617-7598-4aef-b5bd-db9186b8184b}</Project>
+      <Name>Commands.Common.Authentication.Abstractions</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\..\Common\Commands.Common\Commands.Common.csproj">
+      <Project>{5ee72c53-1720-4309-b54b-5fb79703195f}</Project>
+      <Name>Commands.Common</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Common\Commands.ResourceManager.Common\Commands.ResourceManager.Common.csproj">
+      <Project>{3819d8a7-c62c-4c47-8ddd-0332d9ce1252}</Project>
+      <Name>Commands.ResourceManager.Common</Name>
+    </ProjectReference>
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Microsoft.Azure.Management.Search, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
+      <HintPath>..\..\..\packages\Microsoft.Azure.Management.Search.2.0.0\lib\net452\Microsoft.Azure.Management.Search.dll</HintPath>
+    </Reference>
+    <Reference Include="Microsoft.Rest.ClientRuntime, Version=2.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
Please remove the ClientRuntime and ClientRuntime.Azure references, as they are found in common.